# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1620125.html
# Commit: c8f5a8ec70c4
# Full Hash: c8f5a8ec70c4e4fd2aef1defe3d27fb6cd831d3b
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-03-06 09:47:39
# Description:
#   Bug 1620125 - Don't assume mVerticalMetrics has necessarily been initialized before DrawOneGlyph is called. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D65567
# ==============================================================================

diff -r 5b935c9b36ab -r c8f5a8ec70c4 gfx/tests/crashtests/1620125.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1620125.html	Thu Mar 05 18:07:17 2020 +0000
@@ -0,0 +1,12 @@
+<style>
+:root {
+  writing-mode: sideways-lr;
+  padding-right: -1;
+}
+.a {
+  font-style: oblique;
+  list-style-type: simp-chinese-formal;
+}
+<style>X</style>
+<ol class="a">
+<li>